lib/msal-common/src/client/Client.ts

```diff
@@ -31,18 +31,18 @@ export type ResponseStateInfo = {
 
 /**
  * AuthModule class
```

Comment:
Also naming inconsistency in this comment

Reply:
Thanks you. I am adding tests and correcting names, will update in a few hours